.github/workflows/hlint.yml

```diff
@@ -0,0 +1,21 @@
+name: HLint
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types:
+      - opened
+      - synchronize
+jobs:
+  hlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: rwe/actions-hlint-setup@v1
+        with:
+          version: '3.3.6'
+      - uses: rwe/actions-hlint-run@v2
+        with:
+          path: '["category/", "client/", "common/", "duoids/", "hedgehog/", "plugin/"]'
+          fail-on: status
```

Comment:
I'm fine with things either way, but what is the difference between creating multiple workflows or creating one workflow with multiple jobs? E.g., should I move Ormolu to its own workflow too?

Reply:
Hmm, that's my default way, but there's probably no need in this case, because hlint and other CI runs in the same environment under the same triggering conditions. I'll merge them.